codebuild_specs/run_e2e_tests_linux.yml

```diff
@@ -30,12 +30,20 @@ phases:
       - export AMPLIFY_PATH=$CODEBUILD_SRC_DIR/out/amplify-pkg-linux-x64
       - echo $AMPLIFY_DIR
       - echo $AMPLIFY_PATH
+      - npm install -g ts-node
```

Comment:
is this still needed? or was this leftover from debugging/testing

Reply:
I am still getting an errors when I omit it.